orchagent/flex_counter/flex_counter_manager.cpp

```diff
@@ -272,7 +253,7 @@ string FlexCounterManager::getFlexCounterTableKey(
 {
     SWSS_LOG_ENTER();
 
-    return group_name + flex_counter_table->getTableNameSeparator() + sai_serialize_object_id(object_id);
+    return group_name + ":" + sai_serialize_object_id(object_id);
```

Comment:
why getTableNameSeparator was removed and replaced with explici ":'"? previous version is more readable and convinient

Reply:
In the new solution, we do not have `flex_counter_table` anymore.